diff -Naur a/setup.py b/setup.py
--- a/setup.py	2016-04-28 13:18:03.397008583 -0700
+++ b/setup.py	2016-04-28 13:28:04.595779984 -0700
@@ -766,7 +766,7 @@
             else:
                 readline_extra_link_args = ()
 
-            readline_libs = ['readline']
+            readline_libs = ['readline', 'termcap']
             if readline_termcap_library:
                 pass # Issue 7384: Already linked against curses or tinfo.
             elif curses_library:
@@ -1359,7 +1359,7 @@
                 # Bug 1464056: If _curses.so links with ncursesw,
                 # _curses_panel.so must link with panelw.
                 panel_library = 'panelw'
-            curses_libs = [curses_library]
+            curses_libs = [curses_library, 'termcap']
             curses_incs = find_file('curses.h', inc_dirs,
                                     [os.path.join(d, 'ncursesw') for d in inc_dirs])
             exts.append( Extension('_curses', ['_cursesmodule.c'],
@@ -1373,7 +1373,7 @@
             elif (self.compiler.find_library_file(lib_dirs, 'termcap')):
                 curses_libs = ['curses', 'termcap']
             else:
-                curses_libs = ['curses']
+                curses_libs = ['curses', 'termcap']
 
             exts.append( Extension('_curses', ['_cursesmodule.c'],
                                    libraries = curses_libs) )
